src/app/containers/Headings/index.jsx

```diff
@@ -38,7 +39,13 @@ const HeadingsContainer = ({ blocks, type }) => {
 
   return (
     <GridConstrain>
-      <Heading text={text}>{renderText()}</Heading>
+      <ServiceContextConsumer>
```

Comment:
Just being nosey, but you could use the `useContext` hook here, might be nicer :)

Reply:
Nice thought, just giving the hooks doc a quick glance over and
>The current context value is determined by the value prop of the nearest <MyContext.Provider> above the calling component in the tree.  
[source](https://reactjs.org/docs/hooks-reference.html#usecontext)

Worries me slightly because is someone reordered or added a new context inside the `ServiceContext` it wouldn't be obvious why their local setup is failing without prior knowledge of both the current usage of `useContext` and how `useContext` actually works.

Happy to use it if other reviewers agree with @dr3 😄 And if we do I'll document the decision somewhere

Reply:
Ahhhh actual I've misread the docs, ignore my comment above. I hadn't realised they are describing multiple use for a the same context provider.